oci/private/download.bzl

```diff
@@ -118,7 +118,11 @@ def _bazel_download(
         headers = {},
         # passthrough
         **kwargs):
-    return rctx.download(**kwargs)
+    # Passing headers to the downloader is only available as of 7.1
+    if versions.is_at_least("7.1.0", versions.get() or "7.1.0"):
```

Comment:
can you remove the 'or' here?  i don't its a good idea to treat any unknown version as 7.1